### DISCUSSION TITLE
Compile errors on SKR 1.4 Turbo configuration?

### DISCUSSION THREAD

**Discussion by scaramonga** [Jyers/Marlin#1097] [Q&A] [answered]
Hello.

Compiling from source with E3 V2 3x3 BLTouch HS profile on an SKR 1.4 Turbo with TFT35 v3 E3.  Keep getting these 3 errors?

![Annotation 2021-07-01 132605](https://user-images.githubusercontent.com/83183034/124124298-2c56a400-da70-11eb-848f-8bcc279436cb.png)

Any ideas?

**Comment by tome9111991** [comment]
Use default marlin brunch

**Reply by scaramonga** [reply]
Sorry, had lunch, no need for brunch :D  So what we are saying here, is that this firmware (Jyers) does not support motherboards, other than the Creality variants?, which is fine, I just want it clear to all :)

**Comment by smooreace** [comment]
Marlin works fine on BTT mainboards. But its up to you to configure it
properly for the mainboard you decide to use. Jyers FW is Marlin at its
core with tweaks to the display functionality. So yes, it will work fine...
after your configure it properly and compile it

**Reply by scaramonga** [reply]
Thank you for replying.  Would you be so kind as to up your config files for me?  I know its a lot to ask :(, but I'm having major troubles here, not as sharp as I used to be lol.

Just 1.4 Turbo board with 2290 steppers, all pins de-soldered and pushed up for reuse (I hate waste ;)), and don't care about sensor less homing.  I have BLTouch, and TFT v3 E3 screen, and I like all other features on this firmware.

If not, no worries, I understand it all.  Thanking you kindly.

**Reply by creativeG911** [reply]
Hello from Greece! Have you made it? Did you succesfully applied this filware on your skr 1.4 turbo? I have made the same purchase

**Comment by smooreace** [comment, marked as answer]
Im running an official 4.2.2 board. So my files wont help you. But if you
search discussions you should find another rather active
conversation pertaining to the SKR board. Just search "SKR".
